plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreModule.java

```diff
@@ -40,19 +40,19 @@
     protected void setup(Binder binder)
     {
         OptionalBinder.newOptionalBinder(binder, ThriftMetastoreClientFactory.class)
-                .setDefault().to(DefaultThriftMetastoreClientFactory.class).in(Scopes.SINGLETON);
```

Comment:
We don't have a convention to import this statically.

```
$ git grep 'import com.google.inject.Scopes;' | wc -l
129

$ git grep 'import static com.google.inject.Scopes.SINGLETON;' | wc -l
22
```

let's not change it now.
you can offer a change in a separate PR